definitions/region/model_native_regions/witch_v5.1.yaml

```diff
@@ -0,0 +1,38 @@
+# The model documentation for WITCH can be found at doc.witchmodel.org
+- WITCH 5.1:
+  - WITCH 5.1|BRAZIL
```

Comment:
This is the region name how it will be displayed in the Scenario Explorer - you could change it to `WITCH 5.1|Brazil` for easier readability if you want...